sections/dashboard/Overview/Overview.tsx

```diff
@@ -39,9 +48,117 @@ const Overview: FC = () => {
 	);
 	const [activeMarketsTab, setActiveMarketsTab] = useState<MarketsTab>(MarketsTab.FUTURES);
 
+	const { network, walletAddress, synthsMap } = Connector.useContainer();
+
+	const { tokenBalances } = useAppSelector(({ balances, exchange }) => ({
+		balancesMap: balances.balancesMap,
+		tokenList: exchange.tokenList,
+		tokenBalances: balances.tokenBalances,
+		balancesStatus: balances.status,
+	}));
+	// Only available on Optimism mainnet
+	const oneInchEnabled = network.id === 10;
+
+	const isL2 = useIsL2();
+
+	const [exchangeTokens, setExchangeTokens] = useState<any>([]);
+	const [kwentaBalance, setKwentaBalance] = useState(zeroBN);
+
+	useContractRead({
+		address: KWENTA_TOKEN_ADDRESS['10'],
```

Comment:
There is also a Goerli address here so we get by network `const tokenAddress = KWENTA_TOKEN_ADDRESS[network.id] ?? KWENTA_TOKEN_ADDRESS['10']`

And then also replace all the other places we reference KWENTA_TOKEN_ADDRESS['10']

Reply:
1inch token list is only enabled on L2 mainnet and Coingecko is limited to query the Kwenta price at correct network (optimism-ethereum).

Reply:
Ok no problem then